customer.go

```diff
@@ -75,9 +75,10 @@ type CustomerRequest struct {
 }
 
 type CustomerSearchResult struct {
-	XMLName           string      `xml:"customers"`
-	CurrentPageNumber int         `xml:"current-page-number"`
-	PageSize          int         `xml:"page-size"`
-	TotalItems        int         `xml:"total-items"`
-	Customers         []*Customer `xml:"customer"`
+	XMLName           string
+	CurrentPageNumber int
+	TotalItems        int
+	TotalIDs          []string
+	PageSize          int
+	Customers         []*Customer
```

Comment:
Could you organize these fields so they're in the same order as the equivalent `Transaction`, `Subscription` and `CreditCard` types? The consistency is helpful, and it separating the fields relating to a single page, and the fields relating to all of the results makes it easier to reason about.

https://github.com/lionelbarrow/braintree-go/blob/d4e31c11b0086e116fee1d2b9631d634226d245e/transaction.go#L255-L262

https://github.com/lionelbarrow/braintree-go/blob/d4e31c11b0086e116fee1d2b9631d634226d245e/subscription.go#L120-L127

Reply:
Ok, will fix today.